packages/cli/README.md

```diff
@@ -0,0 +1,50 @@
+# Zowe Native Protocol plug-in for Zowe CLI
+
+The Zowe Native Protocol plug-in for Zowe CLI interacts with the Zowe Native Protocol, a native integration with z/OS systems over SSH, allowing developers to interact with mainframe resources directly from their command line.
+
+## Features
+
+- **Dataset Operations**: Create, read, write, and delete datasets
+- **USS File Access**: Create, read, write, and delete files and directories in Unix System Services
+- **Job Management**: Submit, view, cancel, hold, and release jobs on z/OS
+
+## Minimum Requirements
+
+- Zowe CLI v8.0.0 (or newer)
+
+## Installation
+
+Access the latest version of the CLI plug-in from the GitHub Releases page.
+
+Install the CLI plug-in from the tarball:
+
+```
+zowe plugins install dist/zowe-native-proto-cli-*.tgz
```

Comment:
We may not want to assume the `dist` folder if the tgz was downloaded from GitHub
```suggestion
zowe plugins install zowe-native-proto-cli-*.tgz
```

Reply:
Good call, addressed in upcoming commit 👍